test/evergreen/install_gcp_dependencies.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
```

Comment:
I don't tink this comment is necessary, we use this everywhere and I believe one could look for the meaning if needed.

Reply:
Done! [5f6589f](https://github.com/wiredtiger/wiredtiger/pull/8842/commits/5f6589fa932fdb03b4dc0351fae386913ae89d4e)